qulice-findbugs/src/main/java/com/qulice/findbugs/FindBugsValidator.java

```diff
@@ -176,14 +176,22 @@ private String excludes(final Environment env,
      * Creates xml with exclude patterns in findbugs native format.
      * @param excludes Exclude patterns
      * @return XML with findbugs excludes
+     * @todo #350 It's better to build XML using Xembly, but when it's
+     *  added to dependencies, build fails with the message:
+     *  Found duplicate and different classes in
+     *  [org.apache.xmlgraphics:batik-ext:1.7,xml-apis:xml-apis:1.3.04]
+     *  org.w3c.dom.events.DocumentEvent
+     *  org.w3c.dom.events.Event
+     *  org.w3c.dom.events.EventException
      */
     private String generateExcludes(final Iterable<String> excludes) {
-        // @checkstyle StringLiteralsConcatenationCheck (5 lines)
-        return "<FindBugsFilter>\n"
-            + "     <Match>\n"
-            + "       <Class name=\"" + excludes.iterator().next() + "\" />\n"
-            + "     </Match>\n"
-            + "</FindBugsFilter>";
+        final StringBuilder xml = new StringBuilder(75);
+        xml.append("<FindBugsFilter><Match><Or>");
+        for (final String exclude : excludes) {
+            xml.append("<Class name=\"").append(exclude).append("\"/>");
```

Comment:
yeah, this is definitely a bad idea, since `exclude` may be equal to, say, `"hey"` and the entire code will crash. Let's find a way to use Xembly